src/test/skript/tests/syntaxes/expressions/ExprEntities.sk

```diff
@@ -1,11 +1,38 @@
 test "entities in chunk":
 	spawn 10 sheep at spawn of world "world"
+
+	set {_radius center} to location(-24, -60, -24)
+	spawn 10 sheep at {_radius center}
+
+	set {_corner::1} to location(20, -62, 20)
+	set {_corner::2} to location(28, -58, 28)
+	spawn 10 sheep at location(24, -60, 24)
+
 	wait 1 tick
```

Comment:
I don't think this is appropriate for the test to do. Is it necessary?

Reply:
Not completely. It spawns 3 batches of sheep so you can test delete on all 3 versions of the expression but we could just not test delete

Reply:
No, I'm talking about the wait.

Reply:
oh, I have no clue. It was there before. I am assuming it takes a tick for the entities to actually spawn

Reply:
I don't think that's correct though, and it means these sheep are around for all tests that take place during that tick
So I think it should be removed if possible.

Reply:
Makes sense. I will do some testing to confirm